eng/scripts/Language-Settings.ps1

```diff
@@ -14,7 +14,15 @@ function Get-dotnet-PackageInfoFromRepo ($pkgPath, $serviceDirectory, $pkgName)
     $projectData = New-Object -TypeName XML
     $projectData.load($projectPath)
     $pkgVersion = Select-XML -Xml $projectData -XPath '/Project/PropertyGroup/Version'
-    return [PackageProps]::new($pkgName, $pkgVersion, $pkgPath, $serviceDirectory)
+    $sdkType = "client"
+    if ($pkgName -match ".ResourceManager." -or $pkgName -match ".Management.")
```

Comment:
FYI `.` in the regex matches any char not the literal if you want to match the literal you will want `\.`.